packages/@react-aria/overlays/src/useOverlayPosition.ts

```diff
@@ -11,7 +11,7 @@
  */
 
 import {calculatePosition, PositionResult} from './calculatePosition';
-import {HTMLAttributes, RefObject, useCallback, useEffect, useRef, useState} from 'react';
+import {HTMLAttributes, RefObject, useCallback, useLayoutEffect, useRef, useState} from 'react';
```

Comment:
Looks like the SSR tests are failing, think you can use `useLayoutEffect` from the react-aria package to fix